src/dcos_e2e/node.py

```diff
@@ -140,15 +140,20 @@ def popen(
         ssh_args = self._compose_ssh_command(args=args, user=user, env=env)
         return Popen(args=ssh_args, stdout=PIPE, stderr=PIPE)
 
-    def send_file(self, local_path: Path, remote_path: Path) -> None:
+    def send_file(
+        self,
+        local_path: Path,
+        remote_path: Path,
+        user: str,
+    ) -> None:
         """
         Copy a file to this node.
 
         Args:
             local_path: The path on the host of the file to send.
             remote_path: The path on the node to place the file.
+            user: The name of the user used to send the file over SSH.
```

Comment:
This makes it seem like it is a local user. Perhaps describe this as "the remote user". Optionally change `user` to `remote_user` but I don't like that because it conflicts with `run` a bit. Up to you 😄